src/lib/timeseries.js

```diff
@@ -467,24 +466,15 @@ export const computeMonthsAndCO2s = (timeseries, f) => {
   return { months: formatedMonths, CO2s }
 }
 
-export const computeFirstAndLastDay = timeseries => {
-  let firstDay = getStartDate(timeseries[0])
-  let lastDay = getEndDate(timeseries[0])
+export const countUniqDays = timeseries => {
+  const timeserieDays = new Set()
 
   timeseries.forEach(timeserie => {
-    const startDate = getStartDate(timeserie)
-    const endDate = getEndDate(timeserie)
-
-    firstDay = startDate < firstDay ? startDate : firstDay
-    lastDay = endDate > lastDay ? endDate : lastDay
+    const startDate = getStartDate(timeserie).toDateString()
+    timeserieDays.add(startDate)
   })
```

Comment:
That means that if someone's shift is during the night, we'll always count a way/back as 2 days ?
For instance: 

* home -> work -> 20:00
* work -> home -> 03:00

Reply:
Vu avec @BenjaminMty on ne compte pas les déplacements si ceux-ci commence le lendemain moins de 12 heures plus tard qu'un déplacement de la veille. Je corrige et mets la description à jour